doc/changelog.md

```diff
@@ -11,57 +11,69 @@
   @qjit
   @grad
   def f(x):
-    expm = catalyst.accelerate(jax.scipy.linalg.expm)
-    return jnp.sum(expm(jnp.sin(x)) ** 2)
+      expm = catalyst.accelerate(jax.scipy.linalg.expm)
+      return jnp.sum(expm(jnp.sin(x)) ** 2)
   ```
 
   ```pycon
   >>> x = jnp.array([[0.1, 0.2], [0.3, 0.4]])
   >>> f(x)
```

Comment:
I admit, I thought there wasn't any differentiation going on in this example at first! Any issue with having `catalyst.grad` after? Also, would be nice to include these imports (I can't suggest changes 😠)

```
  from catalyst import qjit, grad

  @qjit
  @grad
  def f(x):
      expm = catalyst.accelerate(jax.scipy.linalg.expm)
      return jnp.sum(expm(jnp.sin(x)) ** 2)
```